android/uhabits-core/src/main/java/org/isoron/uhabits/core/ui/widgets/NumericalCheckmarkWidgetBehavior.java

```diff
@@ -0,0 +1,25 @@
+package org.isoron.uhabits.core.ui.widgets;
+
+import android.support.annotation.NonNull;
+
+import org.isoron.uhabits.core.commands.CommandRunner;
+import org.isoron.uhabits.core.commands.CreateRepetitionCommand;
+import org.isoron.uhabits.core.models.Habit;
+import org.isoron.uhabits.core.models.HabitList;
+import org.isoron.uhabits.core.models.Timestamp;
+
+import javax.inject.Inject;
+
+public class NumericalCheckmarkWidgetBehavior extends BaseWidgetBehavior{
```

Comment:
Please remove `NumericalCheckmarkWidgetBehavior` and `BaseWidgetBehavior`. The method `setNumericValue` can be added to `WidgetBehavior` instead.